docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -0,0 +1,229 @@
+
+However, the sample *[Logs] Unique Visitors vs. Average Bytes* chart supports both clicking on a data point and selecting a range. When you create a URL drilldown for this  chart, you have the following choices:
+
+[role="screenshot"]
+image:images/url_drilldown_trigger_picker.png[Trigger picker: single click and range selection]
+
+Variables in the URL template differ per trigger. 
+For example, "single click" has `{{event.value}}` and "range selection" has `{{event.from}}` and `{{event.to}}`. 
+You can create multiple URL drilldowns per panel and attach them to different triggers.
+
+[float]
+[[templating]]
+==== URL templating language
+
+The URL template input uses Handlebars — a simple templating language. Handlebars templates look like regular text with embedded Handlebars expressions.
+
+```
+https://github.com/elastic/kibana/issues?q={{event.value}}
+```
+
+A Handlebars expression is a `{{`, some contents, followed by a `}}`. When the drilldown is executed, these expressions are replaced by values from the dashboard and interaction context.
+
+Refer to Handlebars https://handlebarsjs.com/guide/expressions.html#expressions[documentation] to learn about advanced use cases.
+
+[[helpers]]
+In addition to https://handlebarsjs.com/guide/builtin-helpers.html[built-in] Handlebars helpers, you can use the following custom helpers:
+
+
+|===
+|Helper |Use case |Examples
+
+|json
+|Serialize variables in JSON format
+|`{{json event}}` +
+`{{json event.key event.value}}` +
+`{{json filters=context.panel.filters}}`
+
+
+|rison
+|Serialize context variables in https://github.com/w33ble/rison-node[rison] format. Rison is a common format for {kib} apps for storing state in the URL.
+|`{{rison event}}` +
+`{{rison event.key event.value}}` +
+`{{rison filters=context.panel.filters}}` +
+
+
+|date
+|Format dates. Supports relative dates expressions (for example,  "now-15d"). Refer to the https://momentjs.com/docs/#/displaying/format/[moment] docs for different formatting options.
+|`{{ date event.from “YYYY MM DD”}}` + 
+`{{date “now-15”}}`
+|===
+
+
+[float]
+[[variables]]
+==== URL template variables
+
+The URL drilldown template has three sources for variables:
+
+. *Global* static variables that don’t change depending on the  place where URL drilldown is used or which user interaction executed the drilldown. For example: `{{kibanaUrl}}`.
+. *Context* variables that change depending on where the drilldown is created and used. These variables are extracted from a context of a panel on a dashboard. Example: ``{{context.panel.filters}}`` which gives access to filters that applied to the current panel.
+. *Event* variables that depend on the trigger context. These variables are dynamically extracted from the interaction context when the drilldown is executed.
+
+[[values-in-preview]]
+A subtle but important difference between *context* and *event* variables is that *context* variables use real values in previews when creating a URL drilldown. 
```

Comment:
I'm thinking __event__ might also have some real data in preview in the future, for example the data table, maybe we should rephrase this somehow or omit.

Reply:
This is true and up for discussion in: https://github.com/elastic/kibana/issues/76226
But for now `event` variables are replaced by dummy values in preview, it could cause confusion and I wanted to emphasis this. We will remove this clarification when we improve that UX.